native/src/main/java/io/ballerina/stdlib/oracledb/utils/ConverterUtils.java

```diff
@@ -229,7 +229,7 @@ public static BArray convertByteArrayToBallerinaType(DefaultResultParameterProce
             }
             return byteDataArray;
         } else {
-            throw new UnsupportedTypeError(String.format("Cannot cast varray to type: %s", type));
+            throw new UnsupportedTypeError(String.format("Cannot cast array to type: %s", type));
```

Comment:
Why is the default parameter processor passed to this method? This is not standard practise

Reply:
Moved few methods in https://github.com/ballerina-platform/module-ballerinax-oracledb/pull/32/commits/38efc04030ca7722f72f8743b9f51d1e7219421c